libs/designsystem/carousel/src/carousel.component.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  Component,
+  ContentChild,
+  EventEmitter,
+  Input,
+  Output,
+  TemplateRef,
+  ViewChild,
+} from '@angular/core';
+import { SwiperOptions } from 'swiper';
+import { register } from 'swiper/element/bundle';
+import { UniqueIdGenerator } from '@kirbydesign/designsystem/helpers';
+import { CarouselSlideDirective } from './carousel-slide.directive';
+
+register();
+
+export type SelectedSlide = {
+  slide: any;
+  index: number;
+};
+
+export type SlidesOptions = SwiperOptions;
```

Comment:
Why do we rename this? 
if it is to make sure implementer uses ours, perhaps we could call it something which is closer to the original name:
```suggestion
export type KirbySwiperOptions = SwiperOptions;
```